core/src/resources/kp/entities/kp.entity.ts

```diff
@@ -0,0 +1,24 @@
+import { Project } from 'src/resources/project/entities/project.entity';
+import { KpUnit } from 'src/shared/enums/kp-unit.enum';
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class Kp {
+  @PrimaryGeneratedColumn({ name: 'kp_id' })
+  kpId: number;
+
+  @Column()
+  start: number;
+
+  @Column()
+  end: number;
+
+  @Column({ name: 'kp_unit', type: 'enum', enum: KpUnit, default: KpUnit.METER })
+  kpUnit: KpUnit;
```

Comment:
kpUnit and accuracy  should not be part of kp entity. They are same for all Kps. So source of truth for this data should be one place. Remove kpUnit and accuracy from Kp and add it to project table as projectSetting. In turnkey for example there is Organization Setting under Organization entity. We should do same for project.